src/components/Topnav.tsx

```diff
@@ -1,8 +1,66 @@
-import React from "react";
+import { Avatar, Navbar } from "flowbite-react";
+import LogoSmall from "../assets/LogoWhiteSmall.svg";
+import StudentAvatar from "../assets/StudentAvatar.svg";
+import ProfessorAvatar from "../assets/ProfessorAvatar.svg";
+import { useState } from "react";
+import { CATEGORIES } from "../utils/TopnavConstants";
+import { useLocation } from "react-router";
+import { Link } from "react-router-dom";
 
-// TODO
 function Topnav() {
```

Comment:
not too happy on how big it is..
might move it to the features folder and split it into subcomponents